pages/[id].js

```diff
@@ -75,7 +75,7 @@ export default function Detail() {
       robusta={data.robusta}
       level={data.level}
       provenance={data.provenance}
-      score={data.score}
+      score={data.score.map((rating) => rating.rating)}
```

Comment:
👍 but `rating.rating` is a little bit repetitive: `data.score.map(({rating}) => rating)` should also work and you do not need to think about a better name for your object. :D

Reply:
Did not think about destructuring here. Good call!